### ISSUE TITLE
Move documentation to hydrus

### ISSUE THREAD

**Issue by chrizandr** [HTTP-APIs/documentation-hydrus#12]
Instead of having separate repos for each project, we should add all documentation in the `doc/` path into the hydrus repo. This way we build documentation directly with hydrus.

We should also implement the same things for other repos that need documentation.

**Comment by Mec-iS** [comment]
all autogenerated doc should be with the code. Also the one specific to a repo. The ecosystem-wide documentation should have its own place.

**Comment by chrizandr** [comment]
Right, but this repo doesn't really have ecosystem wide documentation right? I was under the impression that this is only documentation related to hydrus. In which case we should move it to hydrus.
